config/settings.yml

```diff
@@ -625,3 +625,9 @@ genisis:
   service_path: /COVID19Service
   user: TestUser
   pass: bogus
+
+vre:
```

Comment:
I don't love using the initialism in settings as this is a highly-visible place but I understand given we reference the key name a lot. My argument towards using full service name is that when browsing/organizing the settings file as an unfamiliar reader, it's much easier to figure out what all of the seemingly arcane shortnames *are*.